lib/bridgetown-plausible/builder.rb

```diff
@@ -17,21 +17,22 @@ def build
 
       def render
         domain = options.dig(:domain)&.strip
+        server = options.dig(:server)&.strip || "plausible.io"
 
-        tag = if domain
-          markup_for_domain(domain)
+        tag = if domain && server
+          markup_for_snippet(domain, server)
         else
-          Bridgetown.logger.warn "Plausible", "Domain not configured."
-          markup_for_domain("NOT CONFIGURED")
+          Bridgetown.logger.warn "Plausible", "Domain and/or server not configured."
```

Comment:
hehe, yeah, well. "trust your own code" they said...

the whole if block currently isn't really consistent, actually it would not even need to change. i tend to overcheck conditions even though there's hardly a scenario/execution path to lead to an unset 'server' in this case. but you know... computers...

it is probably sufficient to just verify the domain really, like you did:
```rb
tag = if domain
  markup_for_domain(domain, server)
else
  Bridgetown.logger.warn "Plausible", "Domain not configured."
  markup_for_domain("NOT CONFIGURED", server)
end
```